dipy/data/fetcher.py

```diff
@@ -591,6 +597,125 @@ def read_syn_data():
 """
 
 
+def fetch_viz_icons():
+    """ Download icons for visualization
+    """
+    url = 'https://dl.dropboxusercontent.com/u/2481924/'
```

Comment:
Dropbox is not a good place to put data you want to store for the long run. I am happy to put these data on a permanent URL here at UW, if that's helpful.